src/AutoRest.CSharp/Common/Output/Builders/MultipartSerializationMethodsBuilder.cs

```diff
@@ -72,7 +72,7 @@ public static IEnumerable<MethodBodyStatement> BuildMultipartSerializationMethod
                             /*return BinaryData.FromStream(stream);*/
                             Return(BinaryDataExpression.FromStream(stream, false)),
                             /*return new BinaryData(stream.GetBuffer().AsMemory(0, (int)stream.Position));*/
-                            Return(New.Instance(typeof(BinaryData), new[]{ new InvokeInstanceMethodExpression((new StreamExpression(stream)).GetBuffer, "AsMemory", new[] { Literal(0), new CastExpression(stream.Property("Position"), typeof(int)) }, null, false) }))
+                            Return(New.Instance(typeof(BinaryData), new[]{ new InvokeInstanceMethodExpression(new StreamExpression(stream).GetBuffer(), "AsMemory", new[] { Literal(0), new CastExpression(stream.Property("Position"), typeof(int)) }, null, false) }))
```

Comment:
The `GetBuffer` on `Stream` class is a method, therefore our expression should express it as a method even if it takes no arguments.